kubernetes/linera-validator/templates/scylla-config.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: scylla-config
+  namespace: scylla
+data:
+  scylla.yaml: |
+    query_tombstone_page_limit: 200000
```

Comment:
Can we get a newline at the end of this file? :)

Reply:
If you configure your editor to respect `.editorconfig` in the repo this should happen automatically since we have

```
insert_final_newline = true
```